src/components/MDX/MDXComponents.tsx

```diff
@@ -368,8 +370,28 @@ function YouTubeIframe(props: any) {
   );
 }
 
-function Image(props: any) {
-  return <img className="max-w-[calc(min(700px,100%))]" {...props} />;
+function Image(allProps: any) {
+  const {height, width, style, ...props} = allProps;
+  return (
+    <div
+      style={{
+        position: 'relative',
+        width: !width ? '' : `${width}px`,
+        maxHeight: !height ? '' : `${height}px`,
+        maxWidth: 'calc(min(700px,100%))',
+        ...(!style ? {} : style),
+      }}
+      className={styles.imageContainer}>
+      <NextImage
+        {...props}
+        src={
+          props?.src?.startsWith('/') ? props.src : props.src.replace('..', '')
+        }
+        layout={'fill'}
+        className={styles.image}
```

Comment:
can we use tailwind classname here as well?

Reply:
Sure